omnisend/class-omnisend-core-bootstrap.php

```diff
@@ -74,12 +82,14 @@ function ( $user_login, $user ) {
 
 			add_action( OMNISEND_CORE_CRON_SYNC_CONTACT, 'Omnisend\Internal\Sync::sync_contacts' );
 		}
+
+		// self::migrate_options(); fix bug and uncomment
```

Comment:
relevant?

Reply:
yes, it will be need to updated when we will add 7 day notification option